sdk/storage/azure-storage-blobs/inc/ApiViewSettings.json

```diff
@@ -0,0 +1,17 @@
+{
+  "sourceFilesToProcess": null,
+  "additionalIncludeDirectories": [
+    "../../../core/azure-core/inc",
+    "../../azure-storage-common/inc"
+  ],
+  "sourceFilesToSkip": [
+  ],
+  "additionalCompilerSwitches": [],
+  "includeInternal": false,
+  "includeDetail": false,
+  "includePrivate": false,
+  "filterNamespace": "Azure::Storage::Blobs",
```

Comment:
```suggestion
  "filterNamespace": ["Azure::Storage::Blobs", "Azure::Storage::Sas", "Azure::Storage"],
```

Reply:
Adding these fields implies that the files in the azure.storage.blobs package creates types within the `Azure::Storage::Sas` and `Azure::Storage` namespace. 

But it was my understanding that all types in `Azure::Storage` would be defined within azure.storage.common, is that not the case?

I understand that there may be exceptions to this rule today, but the ApiView tooling is designed to reflect the API Guidelines, and I believe that the API guidelines require that all packages in the foo.bar package be contained in the `Foo::Bar` namespace..

Reply:
I agree, but then should we have an APIView and APIViewSettings.json for azure.storage.common, so those APIs also get generated and can be reviewed? If not, which APIView will have the types from the azure storage common package?

I know keyvault shared is done differently than storage common, but will we see the public (and internal) surface area defined in those files in some KeyVault APIView as well, and not filtered out?

Reply:
THe next iteration of this will have an ApiViewSettings for Azure.storage.common. Unfortunately it requires a revision of the APIView tool, so I'm holding off adding it for now.